cypress/integration/login.js

```diff
@@ -8,24 +8,102 @@ describe('Login functionality', function() {
     })
 
     it('Should be able to login with role User', function () {
+        cy.get('[id="loginForm.userId"]').click({force:true})
+        cy.get('[aria-label="TestCon User 17"]').click()
+        cy.get('[id="loginForm.role"]').click({force:true})
+        cy.get('[aria-label="User"]').click()
+        cy.get('[type="submit"]').click()
+
+        cy.get('.user-info__title').contains('TestCon User 17')
+        cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.main-nav').find('.main-nav__link--active').contains('Time Logging').should('have.css', 'color', 'rgb(64, 76, 237)')
+    })
+})
+
+describe('Team Lead role', function() {
+    
+    it('Should display validation for empty user after attempted loggin', function () {
+        cy.visit('/')
+        cy.get('.Select.not-valid').should('not.visible')
+        cy.get('[type="submit"]').click()
+        cy.get('.Select.not-valid').should('be.visible')
+    })
+
+    it('Should be able to login with role Team Lead', function () {
         cy.get('[id="loginForm.userId"]').click({force:true})
         cy.get('[aria-label="TestCon User 17"]').click()
         cy.get('[id="loginForm.role"]').click({force:true})
         cy.get('[aria-label="Team Lead"]').click()
         cy.get('[type="submit"]').click()
 
-        cy.url().should('include', '/time-logging')
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('TestCon User 17')
         cy.get('.main-nav').find('li').should('have.length', 2)
+        cy.get('.main-nav').find('.main-nav__link--active').contains('Time Logging').should('have.css', 'color', 'rgb(64, 76, 237)')
+    })
+})
+
+describe('Manager role', function() {
```

Comment:
Please use full tests descriptions as it is hard to say what does this test test.